src/acvp_build_register.c

```diff
@@ -4535,6 +4538,113 @@ static ACVP_RESULT acvp_build_safe_primes_register_cap(ACVP_CTX *ctx,
     return ACVP_SUCCESS;
 }
 
+static ACVP_RESULT acvp_build_lms_register_cap(ACVP_CTX *ctx,
+                                                       JSON_Object *cap_obj,
+                                                       ACVP_CAPS_LIST *cap_entry) {
```

Comment:
formatting seems odd - tabs and spaces, or just the way git is showing it?

Reply:
All spaces, just too many in a few places somehow. fixed